external/fv3fit/fv3fit/train.py

```diff
@@ -64,7 +60,7 @@ def _get_model(config: fv3fit.TrainingConfig) -> Estimator:
             scaler_kwargs=config.scaler_kwargs,
             **config.hyperparameters,
         )
-    elif isinstance(config, fv3fit.KerasTrainingConfig):
+    elif isinstance(config, fv3fit.DenseTrainingConfig):
```

Comment:
Is this a circular import? Should we import these configs directly?

Reply:
It isn't, but it's close. My plan is to declare these in the module that declares the Estimator, and inject them into config.py (and fv3fit's `__init__.py`) through registration. It's just not possible for this PR because of the hard-codes needed in config.py to handle backwards compatibility translation from the old config object.